CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+# (unreleased)
+
+* Add OnPeerStatusChanged option to NewChannel to receive notifications when
```

Comment:
I think the README shouldn't get into specifics about when the method is called or how it should be handled. That should be part of the API documentation. Ideally this should be 1-2 lines at most